contents/install/new_pod/index.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Install diaspora*
+nav_title: Installation
+subtitle: Build your own home.
+---
+
+**This is a guide for server administrators or developers**, and while we do not want to scare you away, we mean that. If you want to set up a production-ready diaspora\* on a server, you should be a *somewhat* experienced server admin. There is only so much we can cover in this guides, and we will not get into setting up the server itself. This means that things like setting up firewalls, performing regular updates, and doing other things that keep your setup secure is beyond our scope, and you should be very comfortable taking care of that yourself. There is nothing worse than using diaspora\*, but having your data, or even worse - other people's data, leaked, because you ran an outdated OpenSSH version.
```

Comment:
```suggestion
**This is a guide for server administrators or developers**, and while we do not want to scare you away, we mean that. If you want to set up a production-ready diaspora\* on a server, you should be a *somewhat* experienced server admin. There is only so much we can cover in this guides, and we will not get into setting up the server itself. This means that things like setting up firewalls, performing regular updates, and doing other things that keep your setup secure is beyond the scope of these instructions, and you should be very comfortable taking care of that yourself. There is nothing worse than using diaspora\*, but having your data, or even worse - other people's data, leaked, because you ran an outdated OpenSSH version.
```